src/utils/helpers/structure/addApplicationResponses.ts

```diff
@@ -27,8 +27,8 @@ const addApplicationResponses = (
       element.enableViewHistory =
         groupedApplicationResponses.length > 2 ||
         (element.latestApplicationResponse && element.previousApplicationResponse
-          ? element.latestApplicationResponse.value.text !==
-            element.previousApplicationResponse.value.text
+          ? element.latestApplicationResponse?.value?.text !==
```

Comment:
technically speaking, i think it should be `latestApplicationResponse.value?.text` (i.e. first question mark not needed since we check both latestApplicationResponse and previousApplicationResponse not being null/undefined)

Reply:
Okay, sure thing 0418b1e5